pkg/cli/rm.go

```diff
@@ -39,20 +45,23 @@ func (a *Rm) Run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-
-	// If nothing is set default to containers
-	if !(a.Images ||
-		a.Volumes ||
-		a.Secrets ||
-		a.Containers) {
-		a.Containers = true
+	if len(args) == 0 {
+		pterm.Error.Println("No AppName arg provided")
 	}
-
 	if a.All {
-		a.Volumes = true
-		a.Images = true
-		a.Containers = true
-		a.Secrets = true
+		rmObjects = RmObjects{
+			App:    true,
+			Secret: true,
+			Volume: true,
+		}
+	} else if len(a.Type) > 0 { // If nothing is set default to containers
```

Comment:
comment is wrong? if nothing is set you are defaulting to apps, not containers on line 63 (and maybe this comment should go down there?)